### ISSUE TITLE
FLYP still can't rival the standard supervised learning

### ISSUE THREAD

**Issue by jsrdcht** [locuslab/FLYP#2]
Imagenet ACC for FLYP is about 83%, but the supervised counterpart with CE is 85.7% now.

**Comment by SachinG007** [comment]
Hi @jsrdcht,
We understand that you are referring to the modified finetuning strategy adopted in https://arxiv.org/abs/2212.06138, where they mainly train for longer and also use some additional augmentations. 
FLYP uses no augmentations and trains for same epochs (10) as in the related prior work like https://arxiv.org/abs/2109.01903.

Secondly, the modified training routine would still not resolve the ID OOD accuracy tradeoff when doing standard supervised learning.
